package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf static/dist",
     "nodecheckversion": "node ./scripts/node-checkversion.js",
     "preinstall": "npm run nodecheckversion",
-    "postinstall": "sh ./scripts/upstream.sh && npm run build",
```

Comment:
Ah ok. Can maybe just roll-up setting upstream into just the `update` script then.